docs/guides/admin/docs/installation/debs.md

```diff
@@ -77,6 +77,10 @@ accomplished by running the following:
 If you wish to use the upstream OpenSearch repository directly be aware that Opencast only supported with OpenSearch 1.x
 and will not work with OpenSearch 2.x yet.  Future support for this is forthcoming.
 
+To install the ICU plugin for OpenSearch, run the following:
```

Comment:
We should tell the people why we need the ICU Plugin. Currently, this dependency is not justified in the docs.

Reply:
This should also clarify that things *will not work* without this call. I would absolutely skip this as a naive just-testing adopter, and then be confused when it all looks like it should work, but doesn't.